src/server/lib/__tests__/headers.test.ts

```diff
@@ -70,48 +70,17 @@ describe('Content Security Policy headers', () => {
         response.header['content-security-policy'].split(';');
       // Does the CSP header match what we expect it to be? If the CSP settings
       // in Helmet are updated, we expect to need to update these tests.
-      expect(splitCSPHeader).toContain("base-uri 'none'");
+      expect(splitCSPHeader).toContain(`${baseUri} 'none'`);
       expect(splitCSPHeader).toContain("default-src 'none'");
       expect(splitCSPHeader).toContain(
-        [
-          'script-src',
-          baseUri,
-          "'sha256-411aj9j2RJj78RLGlCL/KDMK0fe6OEh8Vp6NzyYIkP4='",
-          'www.google-analytics.com',
-          'sourcepoint.theguardian.com',
-          'gdpr-tcfv2.sp-prod.net',
-          'ccpa.sp-prod.net',
-          'ccpa-service.sp-prod.net',
-          'ccpa-notice.sp-prod.net',
-          'cdn.privacy-mgmt.com',
-          'www.google.com',
-          'www.gstatic.com',
-          'assets.guim.co.uk',
-          "'unsafe-eval'",
-        ].join(' '),
+        `script-src ${baseUri} sourcepoint.theguardian.com gdpr-tcfv2.sp-prod.net ccpa.sp-prod.net ccpa-service.sp-prod.net ccpa-notice.sp-prod.net cdn.privacy-mgmt.com www.google.com www.gstatic.com assets.guim.co.uk 'unsafe-eval'`,
       );
       expect(splitCSPHeader).toContain(
-        `img-src ${baseUri} static.guim.co.uk ophan.theguardian.com www.google-analytics.com www.google.com`,
+        `img-src ${baseUri} static.guim.co.uk ophan.theguardian.com www.google.com`,
```

Comment:
Is the `www.google.com` here modelling the reCAPTCHA script?

Reply:
yup https://github.com/guardian/gateway/blob/3ce27f49fa3aac254b4e02960ca734c36387a750/src/server/lib/middleware/helmet.ts#L15